### PR TITLE
feat: update test

### PR DESCRIPTION
**Pull request by jmiquel**
Updated tech test

### CODE REVIEW

**Review by LefterisHaritou — CHANGES_REQUESTED** [review]
Let's make it only TypeScript. Not JS

File: README.md [hunk 1]
```diff
@@ -4,82 +4,58 @@
...
-##### A sample video of the Buffs showing on the current iOS version can be seen here:
-
-https://github.com/buffup/AndroidTechTest/blob/master/Buff.mov?raw=true
-
+- [ ] When the user selects an answer, the correct answer should be highlighted letting the user if they got it right (UI of your choice)
 
 ## What we are looking for:
 
-- A demo website that hosts the video player and the  Javascript/Typescript library  
+- A demo page with the video player and the Javascript/Typescript file.
```
Comment (LefterisHaritou) [inline README.md:36]: Only Typescript

File: README.md [hunk 2]
```diff
@@ -4,82 +4,58 @@
...
-## Part B: Web Library
-
-We now want a JS/Typescript library distributed as an npm package, that allows to display content (Buff's as we call them) over any existing video player.
-
-The Developer using our library should import our library and initialize it with the ID of the player.
-
-The library then should locate the HTML player frame, and render on top of that HTML content (our Buffs)
-
-### Library Requirements
+We now want a JS/Typescript file that allows to display content (Buff's as we call them) over any existing video player.
```
Comment (LefterisHaritou) [inline README.md]: Only TypeScript

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <section>
+      <video id="buffVideo" controls>
+        <source
+          src="https://buffup-public.s3.eu-west-2.amazonaws.com/video/toronto+nba+cut+3.mp4"
```
Comment (LefterisHaritou) [inline index.html]: Use this Video file instead: 
https://buffup-public.s3.eu-west-2.amazonaws.com/video/Auto+Buff+Stevie+G.mp4

**Review by LefterisHaritou — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (jmiquel) [reply index.html]: Changed

File: README.md (hunk 2 above)
Comment (jmiquel) [reply README.md]: Turned repo into a small app to support TS compilation

**Review by LefterisHaritou — APPROVED** [review]

**Pull request merged**